tst/functions/assertions/HaveParameter.Tests.ps1

```diff
@@ -305,6 +307,38 @@ InPesterModuleScope {
                 $err.Exception.Message | Verify-Equal "Expected command Invoke-DummyFunction to have a parameter ParamWithNotNullOrEmptyValidation, which is mandatory, of type [System.TimeSpan], the default value to be 'wrong value' and has ArgumentCompletion, because of reasons, but it wasn't mandatory, it was of type [System.DateTime], the default value was '(Get-Date)' and has no ArgumentCompletion."
             }
         }
+
+        Context 'Using InParameterSet' {
```

Comment:
We should add a test case when `InParameterSet` is used without `HaveParameter` (plus a test for negate with `Not`). Should it throw for invalid use, or pass when parameter set exist, and fail if parameter set does not exist?

Reply:
`-HaveParameter` is the operator name. It's mandatory so `Should` triggers the right operator-code.